scripts/run_backend_tests_test.py

```diff
@@ -618,11 +619,13 @@ def test_invalid_test_target_message_is_displayed_correctly(self) -> None:
         swapcheck_coverage = self.swap(
             run_backend_tests, 'check_coverage',
             lambda *unused_args, **unused_kwargs: ('', 100.00))
-        with self.swap_execute_task, swapcheck_coverage, self.swap_redis_server:
-            with self.swap_cloud_datastore_emulator, swap_check_results:
-                with self.print_swap:
-                    run_backend_tests.main(
-                        args=['--test_target', 'scripts.run_backend_tests.py'])
+        with self.swap(feconf, 'OPPIA_IS_DOCKERIZED', True):
+            with self.swap_execute_task, swapcheck_coverage:
+                with self.swap_cloud_datastore_emulator, swap_check_results:
+                    with self.print_swap, self.swap_redis_server:
+                        run_backend_tests.main(
+                            args=['--test_target',
+                                  'scripts.run_backend_tests.py'])
```

Comment:
WHy are we changing the existing test? This is changing the test env for `test_invalid_test_target_message_is_displayed_correctly`.

Reply:
Please create seperate test for testing `'OPPIA_IS_DOCKERIZED'`